packages/lesswrong/server/repos/VotesRepo.ts

```diff
@@ -319,4 +320,45 @@ export default class VotesRepo extends AbstractRepo<DbVote> {
         window_start_key;
     `, [postIds, startDate, endDate]);
   }
+
+  /**
+   * Get the ids of all votes where user1 and user2 have voted on the same document. This is mainly
+   * for the purpose of nullifying votes where a user has double-voted from an alt.
+   */
+  async getSharedVoteIds({ user1Id, user2Id }: { user1Id: string; user2Id: string; }): Promise<string[]> {
+    const results = await this.getRawDb().any<{vote_id: string}>(
+      `
+      WITH JoinedVotes AS (
+        SELECT
+          v1._id AS v1_id,
+          v1.power AS v1_power,
```

Comment:
IMO remove the fields that you don't need.

Reply:
I'm going to leave them as I thought it might be useful for debugging